docs/modules/ROOT/pages/install.adoc

```diff
@@ -11,47 +11,57 @@ Perform the following steps:
 
 . Download the change agent tar file from the https://downloads.datastax.com/#cassandra-change-agent[DataStax downloads page].
 
-
 The following files are available in the tar file:
-
-[cols="1,1,1"]
++
+[cols="1,1"]
 |===
-| Cassandra type | Streaming platform |  JAR file
-
-| Apache Cassandra 3.x | Apache Pulsar | agent-c3-pulsar-<version>-all.jar
-| Apache Cassandra 3.x | Luna Streaming | agent-c3-luna-<version>-all.jar
-| Apache Cassandra 4.x | Apache Pulsar | agent-c4-pulsar-<version>-all.jar
-| Apache Cassandra 4.x | Luna Streaming | agent-c4-luna-<version>-all.jar
-| DSE 6.8.16+ | Apache Pulsar | agent-dse4-pulsar-<version>-all.jar
-| DSE 6.8.16+ | Luna Streaming | agent-dse4-luna-<version>-all.jar
+| Cassandra type | JAR file
 
+| Apache Cassandra 3.x | agent-c3-<version>-all.jar
+| Apache Cassandra 4.x | agent-c4-<version>-all.jar
+| DSE 6.8.16+ | agent-dse4-<version>-all.jar
 
 |===
 
-Extract the files from the tar with the following command:
-
+. Extract the files from the tar with the following command:
++
 [source,language-bash]
 ----
 tar xvf cassandra-source-agents-<version>.tar
 ----
 
-Use the file that matches your Cassandra type and streaming platform.
+. Use the file that matches your Cassandra type and streaming platform.
 
 == Start Cassandra with the Change Agent for Cassandra
 
 All data nodes of your Cassandra or DSE datacenter must run the change agent as a JVM agent to send mutations into the events topic of your streaming software.
 Start your Cassandra or DSE nodes with the appropriate producer binary matching your Cassandra (3.11 or 4.0) or DSE (6.8.16) version and your streaming platform (Luna Streaming or Apache Pulsar).
 
-As a Java agent, the change agent parameters take place after the `jar` file name in the form of a comma-separated list of `paramName=paramValue`.
-Here is an example for Luna Streaming/Apache Pulsar on Cassandra version 4.0 nodes:
+In CDC agent versions *before 1.0.3*, the CDC agent Pulsar connection parameters were provided as extra JVM options after the `jar` file name in the form of a comma-separated list of `paramName=paramValue`, as below:
 
-[source,language-bash]
+[source,bash]
 ----
 export JVM_EXTRA_OPTS="-javaagent:/path/to/agent-c4-luna-<version>-all.jar=pulsarServiceUrl=pulsar://pulsar:6650"
```

Comment:
This approach is also supported in newer agent version. The environment value approach is an added extra feature which determines the default value if the java agent options are not specified. 

See discussion here: https://datastax.slack.com/archives/C01DGDWADD2/p1652077363319589